integration/tests/posit/connect/test_content.py

```diff
@@ -37,3 +40,48 @@ def test_content_item_owner_from_include(self):
         item = self.client.content.find_one(include="owner")
         owner = item.owner
         assert owner.guid == self.client.me.guid
+
+    @pytest.mark.skipif(
+        CONNECT_VERSION <= version.parse("2023.01.1"),
+        reason="Python 3.12 not available",
+    )
+    def test_restart(self):
+        # create content
+        content = self.client.content.create(name="example-flask-minimal")
+        # create bundle
+        path = Path(
+            "../../../resources/bundles/example-flask-minimal/bundle.tar.gz"
+        )
+        path = (Path(__file__).parent / path).resolve()
+        bundle = content.bundles.create(str(path))
+        # deploy bundle
+        task = bundle.deploy()
+        task.wait_for()
+        # restart content
+        content.restart()
+        # delete content
+        content.delete()
+
+    @pytest.mark.skipif(
+        CONNECT_VERSION <= version.parse("2023.01.1"),
+        reason="Quarto not available",
+    )
+    def test_refresh(self):
+        # create content
+        content = self.client.content.create(name="example-quarto-minimal")
+        # create bundle
+        path = Path(
+            "../../../resources/bundles/example-quarto-minimal/bundle.tar.gz"
+        )
+        path = (Path(__file__).parent / path).resolve()
+        bundle = content.bundles.create(str(path))
+        # deploy bundle
+        task = bundle.deploy()
+        task.wait_for()
+        # refresh content
+        task = content.refresh()
+        if task:
+            task.wait_for()
+
+        # delete content
+        content.delete()
```

Comment:
I'm still working on adding support for the skipped versions of Connect, but I'll make a separate change. Theres a bit involved to make it work, so it deserves its own pull request.